pkg/node/node.go

```diff
@@ -282,19 +283,15 @@ func NewBee(addr string, swarmAddress swarm.Address, keystore keystore.Service,
 		settlement = pseudosettleService
 	}
 
-	acc, err := accounting.NewAccounting(accounting.Options{
-		Logger:           logger,
-		Store:            stateStore,
-		PaymentThreshold: o.PaymentThreshold,
-		PaymentTolerance: o.PaymentTolerance,
-		EarlyPayment:     o.PaymentEarly,
-		Settlement:       settlement,
-	})
+	pricing := pricing.New(p2ps, logger, o.PaymentThreshold)
```

Comment:
Should `p2ps.AddProtocol()` be called for pricing as well?

Reply:
added